cityofaustin/atd-moped
# Ability to download project list with visible columns only

## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/15304

## Testing
**URL to test:** 
https://deploy-preview-1258--atd-moped-main.netlify.app/moped/ 

**Steps to test:**

1. Go to the projects list view, click on the Download button.
2. A dialog should appear, select that you only want to download **visible** columns. The options dialog should close and the downloading dialog should open.
3. Check the csv you downloaded and make sure it didn't download all the columns but just the ones that you have visible in the list view table.
4. Click the download button again and this time download **all** columns, check the csv that was downloaded and make sure all columns are there not just visible ones.
5. Test removing or adding columns to the list view table and exporting more csvs.
6. Test the cancel button/clicking away.


---
#### Ship list
